book/src/cluster-driver-registrar.md

```diff
@@ -20,9 +20,9 @@ The CSI `cluster-driver-registrar` is a sidecar container that registers a CSI D
 CSI drivers that use one of the following Kubernetes features should use this sidecar container:
 
 * [Skip Attach](skip-attach.md)
-  * For drivers that don't support `ControllerPublishVolume`, this indicates to Kubernetes to skip the attach operation and eliminates the need to deploy the `external-attacher` sidecar (and in some cases, eliminates the need to deploy any "controller" components at all).
+  * For drivers that don't support [`ControllerPublishVolume`](https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerpublishvolume), this indicates to Kubernetes to skip the attach operation and eliminates the need to deploy the `external-attacher` sidecar (and in some cases, eliminates the need to deploy any "controller" components at all).
```

Comment:
re: eliminates the need to deploy any controller components

except the cluster-driver-registrar itself is a controller component...

Reply:
lol, good point -- originally cluster-driver-registrar was part of the node deployed driver registrar. Wonder if we should make that a supported mode in the future. Will reword this in the meantime.